packages/open-next/tsconfig.json

```diff
@@ -3,7 +3,7 @@
   "compilerOptions": {
     "declaration": true,
     "module": "esnext",
-    "moduleResolution": "nodenext",
+    "lib": ["DOM", "ESNext"],
```

Comment:
Nextjs uses the DOM lib for the Headers object, but I'm not sure what's the actual correct lib to load since it was still missing partial types for the iterators. But using "DOM.iterator" complains about other things... zzz